### DISCUSSION TITLE
Over-the-air updates (using Sparkle)

### DISCUSSION THREAD

**Discussion by JohJakob** [gingerbeardman/stapler#22] [Ideas]
It would be neat to implement an over-the-air update solution like [Sparkle](https://sparkle-project.org) to (automatically) check for updates and install them right inside the app.

**Comment by gingerbeardman** [comment]
I agree.

Alternatives:
- release in Mac App Store

Related
- #21 brew support

**Comment by JohJakob** [comment]
Yes, the Mac App Store could be an alternative (or even an additional way of distribution). However, it would result in more work when preparing an update. Updates via Sparkle could even be automated utilising GitHub releases.

In the end, both ways are similarly convenient for users (with the Mac App Store having a slight advantage of not having to open the app to update).
